planemo/commands/cmd_workflow_edit.py

```diff
@@ -23,8 +23,10 @@ def cli(ctx, workflow_identifier, output=None, force=False, **kwds):
 
     kwds["workflows_from_path"] = True
 
+    # TODO this foo seems unimported (also not sure if works when called with id)
+    runnables = for_path(workflow_identifier)
```

Comment:
Before my merge `for_path` (which misses an import I guess) was called on `workflow_path`, but recently all `workflow_path` were replaced by `workflow_identifier` .. so this is likely broken with my commit.

Reply:
Ah, yeah, this was changed even before that. I think I found a less intrusive way to do this now.